src/main/scala/com/github/unchama/seasonalevents/Utl.scala

```diff
@@ -0,0 +1,44 @@
+package com.github.unchama.seasonalevents
+
+import java.time.LocalDate
+
+import org.bukkit.Bukkit
+import org.bukkit.entity.Entity
+import org.bukkit.inventory.ItemStack
+
+import scala.util.Random
+
+object Utl {
+  /**
+   *指定されたEntityがいるLocationに、指定されたitemをドロップする
+   *
+   * @param entity 対象のエンティティ
+   * @param item ドロップさせるItemStack
+   */
+  def dropItem(entity: Entity, item: ItemStack): Unit = {
+    val rate = SeasonalEvents.config.itemDropRate
+    val rand = new Random().nextDouble() * 100
+    if (rand < rate) {
+      // 報酬をドロップ
+      entity.getWorld.dropItemNaturally(entity.getLocation, item)
+    }
+  }
+
+  /**
+   * 指定された年月日を`LocalDate`に変換する
+   * @param year 年（西暦）
+   * @param month 月（算用数字。0埋め不要）
+   * @param daysOfMonth 日（算用数字。0埋め不要）
```

Comment:
0埋めはIntにはそもそもなさそう

Reply:
```suggestion
   * @param month 月
   * @param daysOfMonth 日
```
`Int`なので算用数字であること、0埋め不要であることは自明なのでなくても良さげ